vars/utils.groovy

```
@@ -21,6 +21,19 @@ def getCosaDir(params = [:]) {
     }
 }
 
+// Clone a git repository in the following dir:
+//    dir:               string -- Directory
+//    git_ref:           string -- Git branch
+//    git_uri:           string -- Git URL
+def gitClone(params = [:]) {
+    def path = params.get('dir', "./src");
+    def branch = params.get('git_uri', "main");
+    def repo = params.get('git_ref', "https://github.com/coreos/coreos-assembler.git");
```

Let's drop this default and let it error out if it's unspecified.